docs/How-To-Open-a-Homebrew-Pull-Request.md

```diff
@@ -106,13 +106,28 @@ To make changes on a new branch and submit it for review, create a GitHub pull r
    * If there's a `bottle do` block in the formula, don't remove or change it; we'll update it when we merge your PR.
 5. Test your changes by running the following, and ensure they all pass without issue. For changed formulae and casks, make sure you do the `brew audit` step after your changed formula/cask has been installed.
 
+### Test your formulae-related changes
+
+   ```sh
+   HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <CHANGED_FORMULA>
+   brew test <CHANGED_FORMULA>
+   ```
+
+### Test your cask-related changes
+
+   ```sh
+   HOMEBREW_NO_AUTO_UPDATE=1 HOMEBREW_NO_INSTALL_FROM_API=1 brew install <CHANGED_CASK>
```

Comment:
Question: Why is `HOMEBREW_NO_AUTO_UPDATE=1` needed now and only for cask-related changes?

Reply:
It is to stop brew updating itself while installing a new cask or formula. This option makes sense for both cask and formula.

Reply:
So the worry is that new changes might get pulled down from the remote tap that create conflicts with local formula/cask changes?